packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -1281,6 +1281,53 @@ void main() {
     await checkDragHandleAndColors();
   });
 
+  testWidgets('Drag handle interactive area size at given dragHandleSize', (WidgetTester tester) async {
+    const Color defaultColor=Colors.blue;
+    const Color hoveringColor=Colors.green;
+
+    Widget buildScaffold(GlobalKey scaffoldKey, {Size? dragHandleSize}) {
+      return MaterialApp(
+        theme: ThemeData.light().copyWith(
+          bottomSheetTheme:  BottomSheetThemeData(
+            dragHandleColor: MaterialStateColor.resolveWith((Set<MaterialState> states) {
+              if (states.contains(MaterialState.hovered)) {
+                return hoveringColor;
+              }
+              return defaultColor;
+            }),
+            dragHandleSize: dragHandleSize
+          ),
+        ),
+        home: Scaffold(
+          key: scaffoldKey,
+          body: const Center(child: Text('body')),
+        ),
+      );
+    }
+
+    const Size extendedDragHandleSize = Size(100, 50);
+    
+    final GlobalKey<ScaffoldState> scaffoldKey = GlobalKey<ScaffoldState>();
+    await tester.pumpWidget(buildScaffold(scaffoldKey, dragHandleSize: extendedDragHandleSize));
+
+    showModalBottomSheet<void>(
+      context: scaffoldKey.currentContext!,
+      showDragHandle: true,
+      builder: (BuildContext context) {
+        return const Text('BottomSheet');
+      },
+    );
+
+    await tester.pump(); // bottom sheet show animation starts
+    await tester.pump(const Duration(seconds: 1)); // animation done
```

Comment:
Nit: comments should start with a capital letter and end with a period, as per the style guide.